installer/charts/values.yaml.tpl

```diff
@@ -130,6 +138,52 @@ iam:
   service:
     annotations:
       service.beta.openshift.io/serving-cert-secret-name: {{ $keycloakRouteTLSSecretName }}
+  keycloakCR:
+    namespace: {{ $keycloakNamespace }}
+    name: keycloak
```

Comment:
Attributes like `name` and `adminSecretName` should be set by default in the `tssc-iam` chart’s `values.yaml`. Therefore, in `values.yaml.tpl`, you should only overwrite attributes, relying as much as possible on the default values from each individual Helm chart.